benchmark build failure

As seen on the stackage build server (building with ghc-8.2):

```
Building benchmark 'bench' for xxhash-0.0.2..
[1 of 1] Compiling Main             ( bench/Bench.hs, dist/build/bench/bench-tmp/Main.o )

bench/Bench.hs:17:10: error:
    • No instance for (GHC.Generics.Generic M32.Hash32)
        arising from a use of ‘Control.DeepSeq.$dmrnf’
    • In the expression: Control.DeepSeq.$dmrnf @M32.Hash32
      In an equation for ‘rnf’: rnf = Control.DeepSeq.$dmrnf @M32.Hash32
      In the instance declaration for ‘NFData M32.Hash32’
   |
17 | instance NFData M32.Hash32
   |          ^^^^^^^^^^^^^^^^^
```

--- a/bench/Bench.hs
+++ b/bench/Bench.hs
@@ -9,12 +9,9 @@ import qualified Data.ByteString.Lazy as L
 import qualified Data.Digest.XXHash as XXHash
 import qualified Data.Digest.Adler32 as Adler32
 import qualified Data.Digest.CRC32 as CRC32
-import qualified Data.Digest.Murmur32 as M32
 import qualified Data.Hashable as Hashable
 
 import Criterion.Main
-import Control.DeepSeq
-instance NFData M32.Hash32
 
 main :: IO ()
 main = do

--- a/xxhash.cabal
+++ b/xxhash.cabal
@@ -49,7 +49,6 @@ benchmark bench
                        bytestring,
                        criterion,
                        digest,
-                       murmur-hash,
                        deepseq,
                        hashable,
                        xxhash

